src/components/selfCheckQuestion/index.tsx

```diff
@@ -0,0 +1,852 @@
+import React, { useState } from "react";
+import {
+    Flex,
+    Box,
+    Heading,
+    Circle,
+    Square,
+    Spacer,
+    Input,
+    InputLeftElement,
+    InputRightElement,
+    InputGroup,
+    CloseButton,
+    Button,
+    ButtonGroup,
+    Stack,
+    IconButton,
+    Select,
+    Textarea,
+    useDisclosure,
+    Modal,
+    ModalHeader,
+    ModalOverlay,
+    ModalContent,
+    ModalFooter,
+    ModalBody,
+    ModalCloseButton,
+} from "@chakra-ui/react";
+
+import { ArrowUpIcon, ArrowDownIcon } from "@chakra-ui/icons";
+
+//Self check question card component
+export const SelfCheckQuestionCard: React.FC = ({
```

Comment:
I would recommend adding the props that this component uses as an Interface - would just make finding bugs easier. Should be a quick fix, here's an example for reference https://github.com/uwblueprint/headsupguys/pull/27/files#diff-a77615b501764f7894a235e85ccffdd4513bbabff29289cd6fda94ee0f6df228R14